src/gui/PasswordEdit.cpp

```diff
@@ -119,3 +142,81 @@ void PasswordEdit::autocompletePassword(const QString& password)
         setText(password);
     }
 }
+
+bool PasswordEdit::event(QEvent* event)
+{
+    if (!m_basePasswordEdit) {
+        // check caps lock state only for primary password edits
+        checkCapslockState();
+    }
+    return QLineEdit::event(event);
+}
+
+#include <iostream>
```

Comment:
This include is out of place, on purpose?

Reply:
That was me debugging. :-D